Corona-Warn-App/src/main/res/color/button_contact_diary_person.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Comments are misleading. Also why there are 3 states when you change only one boolean variable `state_checked` ?

Reply:
Agree, should just be default and state_checked true for this to work.

Reply:
Good point, is changed 👍